SurfaceDevCenterManager/Program.cs

```diff
@@ -105,6 +107,7 @@ private static async Task<ErrorCodes> MainAsync(string[] args)
                 { "submissionid=",     "Specify a specific SubmissionId", v => SubmissionId = v },
                 { "shippinglabelid=",  "Specify a specific ShippingLabelId", v => ShippingLabelId = v },
                 { "publisherid=",      "Specify a specific PublisherId", v => PublisherId = v },
+                { "partnerid=",        "Specify PublisherId to send a shipping label instead of Windows Update", v => AnotherPartnerId = v },
```

Comment:
Typo? "Specify **PartnerID** to send..."

Reply:
Yes, need to fix this

Reply:
Fixed